pkg/synopsysctl/cmd_delete.go

```diff
@@ -303,13 +303,13 @@ var deletePolarisReportingCmd = &cobra.Command{
 		// Deploy the Secret
 		err = KubectlDeleteRuntimeObjects(gcpServiceAccountSecrets)
 		if err != nil {
-			return fmt.Errorf("failed to update the gcpServiceAccount Secrets: %s", err)
+			log.Warnf("failed to update the gcpServiceAccount Secrets: %s", err)
```

Comment:
not sure why we're changing this to a warning... also the comment there doesn't match what the comment is saying.  I think this is not needed for reporting